### ISSUE TITLE
How to do trian for scratch with detectron2?

### ISSUE THREAD

**Issue by olramde** [facebookresearch/detectron2#558]
## ❓ How to use Detectron2

Questions like:

1. How to do X with detectron2?
2. How detectron2 does X?

NOTE:

1. If you met any unexpected issue when using detectron2 and wish to know why,
   please use the "Unexpected Problems / Bugs" issue template.

2. We do not answer general machine learning / computer vision questions that are not specific to
	 detectron2, such as how a model works, how to improve your training, or what algorithm/methods can be
	 used to achieve X.

I want to detect components in the Web UI image. However, the Feature Extractor, which is pre-trained with ImageNet, has a very different domain than the Web UI. So I want to learn model from Scratch. I am using a little modification of the tutorial provided in the colab. So, how can I learn object detection model from scratch?

Thank you.

**Comment by ppwwyyxx** [comment]
We have configs that train models from scratch in the model zoo. (just search "scratch" in https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md).

**Comment by dhaivat666** [comment]
@ppwwyyxx I tried this config where we can train from scratch and applied linear learning rate rule for my batchsize, but loss didn't converge and infact increased over time. Also, is there any config where there is no group normalization and trained from scratch?

**Comment by ppwwyyxx** [comment]
We provide configs that work on standard datasets and we do not help with users's own tasks.

There is only one config in detectron2 that trains from scratch, and the paper https://arxiv.org/abs/1811.08883 will help you understand what you need to train from scratch.

**Comment by dhaivat666** [comment]
Thanks. The paper helped. Managed to get the training to work. Though I noticed a surprising thing. The loss was exploding for first few iterations and then training became normal. I guess it could be because of warm-up phase. Thanks for the heads up.
